pandas/core/generic.py

```diff
@@ -963,6 +963,24 @@ def swaplevel(self, i=-2, j=-1, axis=0):
     # ----------------------------------------------------------------------
     # Rename
 
+    # renamer function if passed a dict
+    def _get_rename_function(self, mapper):
+        """
+        Returns a function that will map names/labels, dependent if mapper
+        is a dict, Series or just a function.
```

Comment:
this doesn't need to be here (in the class). is it used in more than 1 place?

Reply:
I have moved it and removed leading underscore